library/include/MiscUtils.h

```diff
@@ -417,8 +417,10 @@ inline std::string join_strings(const std::string &separator, T &items) {
     return ss.str();
 }
 
-DFHACK_EXPORT std::string toUpper(const std::string &str);
-DFHACK_EXPORT std::string toLower(const std::string &str);
+DFHACK_EXPORT char cp437_toupper(char c);
+DFHACK_EXPORT char cp437_tolower(char c);
+DFHACK_EXPORT std::string toUpper(const std::string &str, bool use_cp437 = false);
```

Comment:
I think ideally this should be some other "encoding" parameter, not a boolean. I'll admit that makes the default arguments in the Lua API wrapper harder to implement, though.

Reply:
CP437 is done with a switch statement. What other encodings do we want, and how would they be implemented?

Reply:
You want a separate copy of the function just to avoid a bool?